xformers/ops/memory_efficient_attention.py

```diff
@@ -22,17 +22,47 @@
 
 
 class AttentionMask:
```

Comment:
Nit: I though this attention mask class was temporary until we merged it with the other AttentionMask available? It would be good to make this distinction clear at some point, otherwise it will be very confusing

Reply:
Let's do that as part of another PR as I'm not very familiar with the other attention masks yet